### PR TITLE
New: Implement admin route view

### PR DESCRIPTION
**Pull request by ldkge**
Resolves #

## Description
Add Admin route view

## Proposed Changes

- Add Admin route view

## Screenshots
Screen Shot 2020-06-04 at 6 09 22 PM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/admin/app/styles/navi-admin.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+// navi styles
+@import 'navi-admin/views/index';
```
Comment (kevinhinterlong) [inline packages/admin/app/styles/navi-admin.less:7]: ?
```suggestion
@import 'navi-admin/index';
```

File: packages/admin/addon/templates/admin.hbs [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+        {{#if this.isTabExpanded}}
+            <DenaliTabs class="navi-admin__content-tab" @alignment="vertical" as |Tabs|>
+                {{#each this.tabOptions as |tabItem|}}
+                    <Tabs.Tab class="navi-admin__content-tab-item">
+                        <LinkTo @route={{tabItem.route}}>{{tabItem.title}}</LinkTo>
+                    </Tabs.Tab>
+                {{/each}}
+            </DenaliTabs>
+        {{/if}}
+        <div class="navi-admin__content">
```
Comment (kevinhinterlong) [inline packages/admin/addon/templates/admin.hbs:23]: Duplicated class name

File: packages/admin/addon/templates/admin.hbs [hunk 3]
```diff
@@ -1,2 +1,27 @@
 {{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
-{{outlet}}
+<div class="navi-admin">
+    <DenaliNavbar class="navi-admin__navbar" @isResponsive={{true}} as |Nav|>
+        <Nav.Left as |Left|>
+            <Left.Icon @icon="menu-dialpad" {{on "click" (toggle "isTabExpanded" this)}} />
```
Comment (jkusa) [inline packages/admin/addon/templates/admin.hbs:5]: I wouldn't worry about this now. We need to finalize the nav bar. This most likely will follow the pattern here: 

https://denali.design/docs/2/guides/get-started

Responsive and hamburger menu for small screens.

File: packages/admin/addon/templates/admin.hbs [hunk 4]
```diff
@@ -1,2 +1,27 @@
...
+        <Nav.Right as |Right|>
+            <Right.Icon @icon="notification" />
+            <Right.Icon @icon="user-profile-circle" />
+        </Nav.Right>
+    </DenaliNavbar>
+    <div class="navi-admin__content">
+        {{#if this.isTabExpanded}}
+            <DenaliTabs class="navi-admin__content-tab" @alignment="vertical" as |Tabs|>
+                {{#each this.tabOptions as |tabItem|}}
+                    <Tabs.Tab class="navi-admin__content-tab-item">
```
Comment (jkusa) [inline packages/admin/addon/templates/admin.hbs:17]: Did you try `<Tabs.LinkToTab>`

Comment (ldkge) [reply packages/admin/addon/templates/admin.hbs]: oh updated it with that now!

File: packages/admin/public/robots.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# http://www.robotstxt.org
+User-agent: *
```
Comment (mattgraham1995) [inline packages/admin/public/robots.txt:2]: Can this file be removed?

Comment (ldkge) [reply packages/admin/public/robots.txt]: removed

File: packages/admin/addon/controllers/admin.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+export default class AdminController extends Controller {
+  /**
+   * @property {Array} tabOptions
+   */
+  @tracked tabOptions = ADMIN_TAB_OPTIONS;
+
+  /**
+   * @property {boolean} isTabExpanded
+   */
+  @tracked isTabExpanded: boolean = true;
```
Comment (mattgraham1995) [inline packages/admin/addon/controllers/admin.ts:28]:
```suggestion
  @tracked isTabExpanded = true;
```
Typescript should be able to infer that this is a boolean when you assign true to it.

File: packages/admin/addon/templates/admin.hbs [hunk 7]
```diff
@@ -1,2 +1,27 @@
...
+    </DenaliNavbar>
+    <div class="navi-admin__content">
+        {{#if this.isTabExpanded}}
+        <DenaliTabs class="navi-admin__content-tab" @alignment="vertical" as |Tabs|>
+            {{#each this.tabOptions as |tabItem|}}
+                <Tabs.LinkToTab class="navi-admin__content-tab-item" @route={{tabItem.route}}>
+                    {{tabItem.title}}
+                </Tabs.LinkToTab>
+            {{/each}}
+        </DenaliTabs>
```
Comment (mattgraham1995) [inline packages/admin/addon/templates/admin.hbs:21]: Can this block be indented once more?

File: packages/admin/addon/templates/admin.hbs [hunk 8]
```diff
@@ -1,2 +1,27 @@
...
+        <Nav.Right as |Right|>
+            <Right.Icon @icon="notification" />
+            <Right.Icon @icon="user-profile-circle" />
+        </Nav.Right>
+    </DenaliNavbar>
+    <div class="navi-admin__content">
+        {{#if this.isTabExpanded}}
+        <DenaliTabs class="navi-admin__content-tab" @alignment="vertical" as |Tabs|>
+            {{#each this.tabOptions as |tabItem|}}
+                <Tabs.LinkToTab class="navi-admin__content-tab-item" @route={{tabItem.route}}>
```
Comment (mattgraham1995) [inline packages/admin/addon/templates/admin.hbs:17]: Can we have a test to make sure navigating to the `admin.roles` route works?

**Pull request merged**